test/integration/default/serverspec/default_spec.rb

```diff
@@ -19,3 +19,11 @@
 describe file('/tmp/chef-vault-environment-secret-from-library') do
   its(:content) { should match(/unicorn/) }
 end
+
+describe file('/tmp/chef-vault-items') do
+  its(:content) { should match(/exception raised/) }
```

Comment:
this seems strange; why are we putting "exception raised" in, when it should be a list?

Reply:
Whoops! Corrected.